### PR TITLE
merge iih + mt hand command in one

### PR DESCRIPTION
**Pull request by Alexdoru**
and move it to GTNHlib so that it's available from every dev environement when using client 17

related pr : https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/1051

### CODE REVIEW

**Comment by Alexdoru** [comment]
image

File: src/main/java/com/gtnewhorizon/gtnhlib/commands/ItemInHandCommand.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        }
+        // spotless:on
+    }
+
+    private void printFluidContent(ItemStack itemStack) {
+        if (itemStack.getItem() instanceof IFluidContainerItem tFluidContainer) {
+            FluidStack fluidStack = tFluidContainer.getFluid(itemStack);
+            if (fluidStack != null) {
+                String s = String.format(
+                        "FluidID: [%d], UnlocName: [%s], Name: [%s]",
```
Comment (glowredman) [inline src/main/java/com/gtnewhorizon/gtnhlib/commands/ItemInHandCommand.java:57]: For the item itself, `Unloc.Name` is used, this should be unified (I personally would like to see `Unlocalized Name` but maybe that's to long.

File: src/main/java/com/gtnewhorizon/gtnhlib/commands/ItemInHandCommand.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        if (iCommandSender instanceof EntityClientPlayerMP player) {
+            ItemStack itemStack = player.getCurrentEquippedItem();
+            if (itemStack == null) {
+                addChatMessage("Hand is empty!");
+                return;
+            }
+            GameRegistry.UniqueIdentifier UID = GameRegistry.findUniqueIdentifierFor(itemStack.getItem());
+            addChatMessage(GREEN.toString() + STRIKETHROUGH + "--------------------" + RESET + " Item info " + GREEN + STRIKETHROUGH + "--------------------");
+            addChatMessage(String.format(GREEN + "Unloc.Name:" + RESET + " [%s]", itemStack.getUnlocalizedName()));
+            if (UID != null) addChatMessage(String.format(GREEN + "ItemName:" + RESET + " [%s]", UID));
```
Comment (glowredman) [inline src/main/java/com/gtnewhorizon/gtnhlib/commands/ItemInHandCommand.java:42]: This would be a good oportunity to use `Registry Name` instead.

**Review by glowredman — COMMENTED** [review]
One enhancement could be to use ChatComponents such that, when clicking on any of the values, that values is copied to the clipboard. The current behaviour (MT snippet being copied to clipboard) should remain though.

In any case, this is a practical addition and long overdue.

**Pull request merged**